stdlib/xml/dom/minidom.pyi

```diff
@@ -87,7 +86,7 @@ class NamedNodeMap:
     def keys(self): ...
     def keysNS(self): ...
     def values(self): ...
-    def get(self, name: str, value: Any | None = ...): ...
+    def get(self, name, value: Incomplete | None = ...): ...
```

Comment:
Any reason you removed the `name` annotation?

Reply:
It was a bad merge, I already fixed it :)